cicd/gitops/viewing-argo-cd-logs-using-the-kibana-dashboard.adoc

```diff
@@ -0,0 +1,14 @@
+:_content-type: ASSEMBLY
+[id="viewing-argo-cd-logs-using-the-kibana-dashboard"]
+= Viewing Argo CD logs using the Kibana dashboard
+include::_attributes/common-attributes.adoc[]
+:context: viewing-argo-cd-logs-using-the-kibana-dashboard
```

Comment:
I suggest removing the `using-the-kibana-dashboard` part:
```suggestion
[id="viewing-argo-cd-logs"]
= Viewing Argo CD logs
include::_attributes/common-attributes.adoc[]
:context: viewing-argo-cd-logs
```
Please note that it's also in the name of this file ^.